packages/components/search/src/components/Feedback.vue

```diff
@@ -0,0 +1,40 @@
+
+<template >
+  <!-- <i18n src="../locales/index.json"></i18n> -->
```

Comment:
remove all commented code and unused code.

Reply:
Removed!